etc/c/odbc/sql.d

```diff
@@ -67,17 +67,17 @@ enum
     SQL_HANDLE_DESC                 = 4,
 
     // * environment attribute *
-    SQL_ATTR_OUTPUT_NTS             = 10001,
+    SQL_ATTR_OUTPUT_NTS             = 10_001,
```

Comment:
tbh, I find it disturbing that we'll change constants in a C binding just to satisfy a style checker. I'd much rather disable the check for C binding files so copy/paste continues to work since you aren't supposed to actually be reading these numbers anyway.